Packs/EWS/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "EWS",
     "description": "Exchange Web Services and Office 365 (mail)",
     "support": "xsoar",
-    "currentVersion": "1.6.1",
+    "currentVersion": "1.7.0",
```

a bug fix should be a revision, not minor
```suggestion
    "currentVersion": "1.6.2",
```